pkgs/python_ot/default.nix

```diff
@@ -8,21 +8,25 @@
   lib,
   ...
 }: let
-  # This overlay is generated by poetry2nix
-  poetry-git-overlay = import ./poetry-git-overlay.nix {inherit pkgs;};
+  workspace = inputs.uv2nix.lib.workspace.loadWorkspace {workspaceRoot = ./.;};
```

Comment:
Add opentitan as a non-flake input, and reference the pyproject there directly?

Reply:
This can't be done right now as OT doesn't have a uv.lock.